### ISSUE TITLE
angular/components#7835 is not going to happen, please update README.md

### ISSUE THREAD

**Issue by Hermholtz** [bithost-gmbh/ngx-mat-select-search#398]
In README.md there's this:
> Important Note: This project is meant as a temporary implementation of angular/components#5697. The goal is to have an implementation in the official Angular Material repository, once angular/components#7835 is merged.

This is not going to happen. angular/components#7835 has been scrapped on Jun 13, 2019, and angular/components#5697 has removed the "under consideration" label in June this year and no activity happened since then.

So ngx-mat-select-search has become the de facto standard component implementing dropdown with search for Angular.

Therefore you should remove the sentences quoted above. Obviously, after over 5 years we can conclude that it's never going to be implemented by the Angular team.

**Comment by macjohnny** [comment]
Thanks for your feedback.
While I think that it would be nice to have https://github.com/angular/components/issues/5697 implemented in angular material itself, the statement you quoted still holds true, this library is intended to be a solution while its not implemented in angular material. So for now, I would keep this quote. Nevertheless, we make sure that ngx-mat-select-search offers a high-quality solution, and I appreciate contributions to the project.

**Comment by melroy89** [comment]
Too bad that https://github.com/angular/components/pull/7835 is indeed closed and not merged. I really believe Angular is not listening to their customers. Look at the amount of likes/icons on that PR.

Maybe update the README and only point to: https://github.com/angular/components/issues/5697 for now.
